search-page-navbar.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"
+    />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Sights & Sounds | Your Event Planner</title>
+  </head>
+  <body>
+    <div class="header">
+      <h1 class="center-align">Sights & Sounds | Your Event Planner</h1>
+    </div>
+    <nav>
+      <div class="nav-wrapper row">
+        <form>
+          <div class="input-field col s2">
+            <input id="search" type="search" required />
+            <label class="label-icon" for="search"
+              ><i class="material-icons">search</i></label
+            >
+          </div>
+          <div class="input-field col s2">
+            <select>
+              <option value="" disabled selected>Choose a Country</option>
```

Comment:
Would be really good in the future if these were dynamically created so the html file was less bloated 😱